.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+# Description
+
+<!-- Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+<!-- If your pull request is not related to a particular issue, delete the statement below. -->
+Closes #(issue) 
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Enhancement (improvement to an already existing feature)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
Line 14 and 25 seem redundant I think. Each PR that requires a docs update should just do it as it's in the checklist.